presto-main/src/main/java/io/prestosql/connector/system/jdbc/FilterUtil.java

```diff
@@ -36,7 +38,7 @@ private FilterUtil() {}
         }
 
         Object value = domain.getSingleValue();
-        return Optional.of(((Slice) value).toStringUtf8());
+        return Optional.of(((Slice) value).toStringUtf8().toLowerCase(ENGLISH));
```

Comment:
This is not correct. This method is not dedicated to table/column names, so should not lower case.
Even if it was dedicated, lowercasing would be misleading. (a non-lowercase expected value never matches, when lowercased will)

Reply:
You are faster than GHA. 
Fixed - moved lowercasing to a different place.